docs/config-yml-settings.md

```diff
@@ -186,13 +184,14 @@ Connect to your cars sensors for accurate data:
 Control how your battery behaves during car charging:
 
 - **car_charging_from_battery** - When True the car can drain the home battery, Predbat will manage the correct level of battery accordingly. When False home battery discharge will be prevented when your car charges, all load from the car and home will be from the grid. This is achieved by setting the discharge rate to 0 during car charging and to the maximum otherwise, hence if you turn this switch Off you won't be able to change your discharge rate outside Predbat. The home battery can still charge from the grid/solar in either case. Only use this if Predbat knows your car charging plan, e.g. you are using Intelligent Octopus or you use the car slots in Predbat to control your car charging.
-    - CAUTION: If you turn this switch back on during a car charging session you will need to set your battery discharge rate back to maximum manually.
+
+  - CAUTION: If you turn this switch back on during a car charging session you will need to set your battery discharge rate back to maximum manually.
```

Comment:
The proper Markdown spec is to have 4 spaces (or a tab) for an indented list, I think you've changed this plus some others to 2.

I went through them all when I redid the docs and set them all to 4 (GitHub was fairly forgiving, and would work with 2).

And if it did with with mkdocs, that'll be because fairly late on I added the sane lists extension, as one of the original lists wasn't working with the strict implementation of Markdown spec.